libecho/src/BitSender.cc

```diff
@@ -0,0 +1,69 @@
+#include "BitSender.h"
+
+#include <cmath>
+
+const QAudioFormat::Endian OUTPUT_BYTEORDER = QAudioFormat::LittleEndian;
+const int OUTPUT_CHANNEL_COUNT = 1;
+const char *OUTPUT_CODEC = "audio/pcm";
+const int OUTPUT_SAMPLERATE = 44100;
+const int OUTPUT_SAMPLESIZE = sizeof(SampleType);
+const QAudioFormat::SampleType OUTPUT_SAMPLETYPE = QAudioFormat::SignedInt;
+/**< QAudioFormat settings we use while sending audio. */
+
+static constexpr size_t SOUNDS_PER_BYTE =
+    8; /**< States the number of sounds we need to play to encode a byte of data. */
+
+void BitSender::send(const std::vector<uint8_t> &buffer) {
+    auto encoded = encode(buffer);
+    output->startStream();
```

Comment:
We discussed moving startStream/stopStream to constructor/destructor. What do you think about it?

Reply:
kki